### PR TITLE
Automated cherry pick of #54459

### PR DESCRIPTION
**Pull request by andyzhangx**
Cherry pick of #54459 on release-1.7.

#54459: change default kind value of azure disk pv

### CODE REVIEW

**Comment by andyzhangx** [comment]
@rootfs thx

**Comment by andyzhangx** [comment]
@rootfs

**Comment by andyzhangx** [comment]
@wojtek-t

File: pkg/volume/azure_dd/azure_common.go [hunk 1]
```diff
@@ -120,7 +121,7 @@ func normalizeFsType(fsType string) string {
 
 func normalizeKind(kind string) (v1.AzureDataDiskKind, error) {
 	if kind == "" {
-		return v1.AzureDedicatedBlobDisk, nil
+		return defaultAzureDiskKind, nil
```
Comment (wojtek-t) [inline pkg/volume/azure_dd/azure_common.go]: I don't understand this change. This seems like no-op to me.

**Pull request closed without merging**